api/src/pcapi/connectors/beneficiaries/educonnect/educonnect_connector.py

```diff
@@ -44,7 +40,12 @@ def get_saml_client() -> Saml2Client:
     config = {
         "entityid": PASS_CULTURE_IDENTITY_ID,
         "metadata": {
-            "local": [EDUCONNECT_METADATA],
+            "local": [
+                path.join(
+                    BASEDIR,
+                    f"{FILES_DIR}/educonnect-metadata/educonnect.{'production' if settings.IS_PROD else 'pr4'}.metadata.xml",
+                )
+            ],
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏On peu squash avec le commit précédent